src/main/scala/akka/wamp/package.scala

```diff
@@ -55,7 +55,7 @@ package object wamp {
 
   final object Uri {
     // strict URI check disallowing empty URI components
-    val regex = "^([0-9a-z_]+\\.)*([0-9a-z_]+)$".r
+    val regex = "^([0-9a-zA-Z_]+\\.)*([0-9a-zA-Z_]+)$".r
```

Comment:
There's an ongoing discussion on the wamp-proto GitHub repository

https://github.com/wamp-proto/wamp-proto/issues/115